lib/util.js

```diff
@@ -38,15 +38,15 @@ const inspectDefaultOptions = Object.seal({
   breakLength: 60
 });
 
-const CIRCULAR_ERROR_MESSAGE = 'Converting circular structure to JSON';
+const CIRCULAR_ERROR_MESSAGE = /Circular/i;
 
 var Debug;
 
 function tryStringify(arg) {
   try {
     return JSON.stringify(arg);
   } catch (err) {
-    if (err.name === 'TypeError' && err.message === CIRCULAR_ERROR_MESSAGE)
+    if (err.name === 'TypeError' && err.message.match(CIRCULAR_ERROR_MESSAGE))
```

Comment:
I think between the current regexp and @cjihrig's suggestion, I would vote for the latter (followed by an assert to make sure the value is non-empty during startup).

The regexp might match a `TypeError` that includes the name of a property named 'circular'.